Modules/rat_rescue.py

```diff
@@ -745,7 +745,8 @@ def rats(self, value):
         else:
             raise TypeError(f"expected type list got {type(value)}")
 
-    async def add_rat(self, name: str = None, guid: UUID or str = None, rat: Rat = None) -> None:
+    async def add_rat(self, name: str = None, guid: UUID or str = None,
```

Comment:
grr, while this would take more vertical space methinks each param should be on a newline. the place you choose to split is just awkward (nitpick)